doc/documentation.rst

```diff
@@ -2,7 +2,7 @@
 
   <div class="container-index">
 
-Documentation of scikit-learn 0.19.dev0
+Documentation of scikit-learn |version|
```

Comment:
> using sphinx magic |version|

Would you have a reference on how this works? Couldn't find it in sphinx docs...

Reply:
Like many things in sphinx I stumbled upon it by accident, so now I can't remember how to get back there!

Reply:
http://www.sphinx-doc.org/en/stable/config.html#confval-version hints at it, but I know it's elsewhere too